client/components/documents.vue

```diff
@@ -69,10 +69,10 @@
           this.documentCreationInProgress = false;
 
           if (error) {
-            Snackbar.enqueue(`Error creating a new document: ${error}`, 'error');
+            Snackbar.enqueue(`${this.$gettext("Error creating a new document")}: ${error}`, 'error');
```

Comment:
This is problematic. You are here assuming that all language will be of format `<error description>: <error>`, but for example in French there is space before `:`, so it should be `<error description> : <error>`. So always when you have interpolation you have to make it part of the string being translated.